contrib/pyln-client/pyln/client/plugin.py

```diff
@@ -974,11 +974,10 @@ def verify_bool(d: Dict[str, JSONType], key: str) -> bool:
             return self._exec_func(self.child_init, request)
         return None
 
-    def _set_config(self, **_) -> None:
+    def _set_config(self, config: str, val: Optional[Any]) -> None:
         """Called when the value of a dynamic option is changed
-        For now we don't do anything.
         """
-        pass
+        self.options[config]['value'] = val
```

Comment:
Hm, it'd be cute if we also had a `threading.Condition` that gets `notify_all` when the value changes. I'll see if I can just hack it up in a separate PR :thinking: